Dockerfile

```diff
@@ -4,28 +4,82 @@ FROM continuumio/miniconda3
 # ENVIRONMENT #
 ###############
 ENV  TETHYS_HOME="/usr/lib/tethys" \
+     TETHYS_PERSIST="/var/lib/tethys_persist" \
+     APPS_ROOT="${TETHYS_PERSIST}/apps" \
```

Comment:
Let's call this `TETHYS_APPS_ROOT` and put it in `${TETHYS_HOME}` instead. directory